torch/fx/graph_module.py

```diff
@@ -194,7 +194,9 @@ def __reduce__(self):
     def __deepcopy__(self, memo):
         fake_mod = torch.nn.Module()
         fake_mod.__dict__ = copy.deepcopy(self.__dict__)
-        return GraphModule(fake_mod, self.graph)
+        graph_module = GraphModule(fake_mod, self.graph)
+        graph_module.__dict__ = fake_mod.__dict__
```

Comment:
Does this contain generated artifacts (`.code` or `.forward`)? If so, you should delete tehm

Reply:
it contains:
dict_keys(['training', '_parameters', '_buffers', '_non_persistent_buffers_set', '_backward_hooks', '_forward_hooks', '_forward_pre_hooks', '_state_dict_hooks', '_load_state_dict_pre_hooks', '_modules', '_graph', 'code', 'attr'])

will delete _modules, _graph and code.